File: CreationalPatterns/Builder/Java/Builder/Implementations/PeppyPannerPizzaBuilder.java
package Builder.Implementations;

import Builder.Interface.PizzaBuilder;
import Models.Pizza;

public class PeppyPannerPizzaBuilder implements PizzaBuilder {

    private Pizza.Builder pizzaBuilder;

    public PeppyPannerPizzaBuilder(){
        this.pizzaBuilder = new Pizza.Builder("Fresh Pan", "Korean");
    }

    @Override
    public void buildCheese() {
        pizzaBuilder.withCheese("Mozzarella");
    }

    @Override
    public void buildToppings() {
        pizzaBuilder.withTopping("Panner");
        pizzaBuilder.withTopping("Red Pepper");
    }

    @Override
    public Pizza getResult() {
        return pizzaBuilder.build();
    }
    
}

File: CreationalPatterns/Builder/C#/Builder/Builders/Implementations/PeppyPannerPizzaBuilder.cs
public class PeppyPannerPizzaBuilder : PizzaBuilder {

    private Pizza.Builder pizzaBuilder;

    public PeppyPannerPizzaBuilder(){
        this.pizzaBuilder = new Pizza.Builder("Fresh Pan", "Korean");
    }

    public void BuildCheese() {
        pizzaBuilder.WithCheese("Mozzarella");
    }

    public void BuildToppings() {
        pizzaBuilder.WithTopping("Panner");
        pizzaBuilder.WithTopping("Red Pepper");
    }

    public Pizza GetResult() {
        return pizzaBuilder.Build();
    }
    
}
